ui/src/components/Log/GoToLineButton.jsx

```diff
@@ -59,11 +59,11 @@ export default class GoToLineButton extends Component {
         <Button
           spanProps={{ className }}
           tooltipProps={{ title: 'Go to Line' }}
-          variant="round"
+          variant="text"
           color="secondary"
           onClick={this.handleOpenClick}
           {...props}>
-          <NumericIcon />
+          <NumericIcon size={20} />
```

Comment:
@helfi92 18px looked too small, so I chose 20. Let me know if we should use 18px.